app/models/comfy/cms/page.rb

```diff
@@ -90,4 +90,14 @@ def ever_been_published?
   def publishable?
     keywords.all?(&:publishable?)
   end
+
+  def published_at
+    if published?
+      updated_at
+    elsif scheduled?
+      scheduled_on
```

Comment:
After re-reading you comment. The requirement is for a `published_at` timestamp. This would be covered if an article has been scheduled